plugins/language_st.lua

```diff
@@ -0,0 +1,125 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+-- Language syntax references
+-- https://pdhonline.com/courses/e334/e334content.pdf
+
+syntax.add {
+  name = "PLC Structured Text IEC 61131-3",
+  files = { "%.stx?$", "%.iecst$" },
+  comment = "//",
+  block_comment = { "%(%*", "%*%)" },
+  patterns = {
+    { pattern = "//.*",                                     type = "comment"  }, -- Single-line comment
+    { pattern = { "%(%*", "%*%)" },                         type = "comment"  }, -- Multi-line comment
+    { pattern = { '"', '"', '\\' },                         type = "string"   }, -- String, quotation marks
+    { pattern = { "'", "'", '\\' },                         type = "string"   }, -- String, apices
+    { pattern = "%w+#[0-9]+m?s?",                           type = "number"   }, -- Time/Date formats
+    { regex   = [[(?>[^(\W]+)(?=\()]],                      type = "function" }, -- Function
```

Comment:
Could this be a simple "standard" function pattern?

Reply:
Yes, probably.